### PR TITLE
added field "create write"

### PR DESCRIPTION
**Pull request by Evgen2325**

### CODE REVIEW

File: .idea/vcs.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (samodur69) [inline .idea/vcs.xml:1]: .idea should be added to gitignore and not be pushed. Major issue

File: myapp/bill/models.py [hunk 2]
```diff
@@ -7,8 +7,8 @@ class Bills(models.Model):
     date = models.DateTimeField('Дата')
 
     def __str__(self):
-        return self.title
+        return f' {self.title} {self.price}р. {self.date}'
 
     class Meta:
```
Comment (samodur69) [inline myapp/bill/models.py:12]: I think its not relevant name for this model

File: myapp/bill/templates/bill/bills.html [hunk 3]
```diff
@@ -6,8 +6,11 @@
 {% block content %}
     <div class="features">
         <h1>Расчет</h1>
-
-        {{ bills }}
+        {% for el in bill %}
+            <div class="alert">
```
Comment (samodur69) [inline myapp/bill/templates/bill/bills.html:10]: Why class name Alert?

File: myapp/bill/templates/bill/bills.html [hunk 4]
```diff
@@ -6,8 +6,11 @@
...
+                <h3>{{ el }}</h3>
```
Comment (samodur69) [inline myapp/bill/templates/bill/bills.html:11]: h3 use for titles or subtitles

**Review by samodur69 — CHANGES_REQUESTED** [review]

File: myapp/bill/templates/bill/bills.html (hunk 4 above)
Comment (Evgen2325) [reply myapp/bill/templates/bill/bills.html:11]: for titles

File: myapp/bill/templates/bill/bills.html (hunk 3 above)
Comment (Evgen2325) [reply myapp/bill/templates/bill/bills.html:10]: it's a design decision) usually that`s use for notifications

File: myapp/bill/models.py (hunk 2 above)
Comment (Evgen2325) [reply myapp/bill/models.py:12]: yes, i`m agreed
and changed name

File: .idea/vcs.xml (hunk 1 above)
Comment (Evgen2325) [reply .idea/vcs.xml:1]: I'll fix it in the last chapter

Comment (samodur69) [reply .idea/vcs.xml:1]: wrong decision

File: myapp/bill/templates/bill/bills.html (hunk 3 above)
Comment (samodur69) [reply myapp/bill/templates/bill/bills.html:10]: here you display elements from list, not alerts. 
Also you has not any alerts css. Then its unnecessary.

File: myapp/bill/templates/bill/bills.html (hunk 4 above)
Comment (samodur69) [reply myapp/bill/templates/bill/bills.html:11]: h1 for titles

Comment (Evgen2325) [reply myapp/bill/templates/bill/bills.html:11]: yes, agreed, thanks!

File: myapp/bill/models.py [hunk 5]
```diff
@@ -7,7 +7,7 @@ class Bills(models.Model):
     date = models.DateTimeField('Дата')
 
     def __str__(self):
-        return f' {self.title} {self.price}р. {self.date}'
+        return f' {self.title}           {self.price}р.           {self.date}'
```
Comment (samodur69) [inline myapp/bill/models.py:10]: what it is????

File: myapp/bill/forms.py [hunk 6]
```diff
@@ -0,0 +1,24 @@
+from .models import Bills
+from django.forms import ModelForm, TextInput, DateTimeInput
+
+
+class BillsForm(ModelForm):
+    class Meta:
```
Comment (samodur69) [inline myapp/bill/forms.py:6]: What mean Meta here?

File: myapp/bill/templates/bill/create.html [hunk 7]
```diff
@@ -7,10 +7,11 @@
     <div class="features">
         <h1>Здесь можно добавить запись</h1>
         <form method="post">
-            <input type="text" placeholder="Название" class="form-control"><br>
-            <input type="text" placeholder="Цена" class="form-control"><br>
-            <input type="date" class="form-control"><br>
-            <input type="time" class="form-control"><br>
+            {% csrf_token %}
+            {{ form.title }}<br>
```
Comment (samodur69) [inline myapp/bill/templates/bill/create.html:11]: <br> is bad idea

**Pull request merged**